### PR TITLE
Override default maximum size of HTTP headers

### PR DESCRIPTION
**Pull request by ker-an**
Node.js 8.14.0 introduced a maximum size for HTTP headers of 8KB. This ended up breaking a lot of our apps because our header sizes are much larger. In Node.js 8.15.0, you can specify a different maximum size using the `--max-http-header-size` option.

While adding a smoke test to next-router to test the maximum header size, we found that the test wasn't passing locally because of `nht run`. Turns out we need to add the `node --max-http-header-size=80000` option when starting node and that's what this PR does!

Resolves https://github.com/Financial-Times/n-heroku-tools/issues/552

### CODE REVIEW

**Comment by ker-an** [comment]
This PR will break any apps that are pinned to Node.js v8.13.0. 

I am blocked from merging this until we resolve https://github.com/Financial-Times/next-transformations/issues/31.

File: tasks/run.js [hunk 1]
```diff
@@ -101,6 +101,16 @@ function runScript (opts) {
 		if (opts.subargs) {
 			args = args.concat(opts.subargs.replace(/^\[/, '').replace(/]$/, '').split(','));
 		}
+
+		/**
+		 * The default maximum size of HTTP headers is 8KB.
```
Comment (adambraimbridge) [inline tasks/run.js]:
```suggestion
		 * Since node version 8.14, the default maximum size of HTTP headers is 8KB.
```
